assets/pages/weekly-subscription-landing/weeklySubscriptionLanding.jsx

```diff
@@ -73,18 +73,35 @@ const content = (
       footer={<Footer />}
     >
       <ProductPagehero
-        headline="Become a Guardian Weekly subscriber"
```

Comment:
this was the only instance of `headline` being used, worth checking with the designers (@blongden73 commented yesterday that he wanted to tweak its colour too) if this element will reappear again in any headers or we can remove it entirely